source/solvers/heat_transfer_scratch_data.cc

```diff
@@ -104,6 +104,8 @@ HeatTransferScratchData<dim>::enable_vof(const FiniteElement<dim> &fe,
   viscosity_1                      = std::vector<double>(n_q_points);
   grad_specific_heat_temperature_1 = std::vector<double>(n_q_points);
 
+  viscous_dissipation_indicator =
+    Parameters::DissipationIndicator::fluid1; // TODO take from parameters
```

Comment:
@blaisb Would you have an insight about how you wish we access this parameter?

I have considered: 
- creating a getter method (in multiphysics_interface or in vof?), 
- modifying the `HeatTransferAssemblerViscousDissipationVOF` specialized method (but then I have to add an optional parameter to the overriden methods in `HeatTransferAssemblerBase`)
- leaving it as is (dissipation in `fluid1` only), knowing that if we need to apply it differently a portion of the code already exists

I would be happy to have your feedback on that :)